src/components/ActionsMenu/ActionsMenu.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import styles from './ActionsMenu.st.css';
+import { TPAComponentsConsumer } from '../TPAComponentsConfig';
+import { ActionsMenuItem } from './Item/ActionsMenuItem';
+import { ACTIONS_MENU__DATA_KEYS } from './dataHooks';
+import { Divider } from '../Divider';
+
+export enum Alignment {
+  left = 'left',
+  center = 'center',
+  right = 'right',
+}
+
+export interface ActionsMenuProps {
+  alignment?: Alignment;
+}
+
+interface DefaultProps {
+  alignment: Alignment;
+}
+
+function getDataAttributes(mobile: boolean) {
+  return {
+    [ACTIONS_MENU__DATA_KEYS.mobile]: mobile,
+  };
+}
+
+/** ActionsMenu */
+export class ActionsMenu extends React.Component<ActionsMenuProps> {
+  static Item = ActionsMenuItem;
+  static Divider = Divider;
+
+  static displayName = 'ActionsMenu';
+  static defaultProps: DefaultProps = {
+    alignment: Alignment.left,
+  };
+
+  render() {
+    const { alignment, children, ...rest } = this.props;
+    return (
+      <TPAComponentsConsumer>
+        {({ mobile }) => (
+          <ul
+            key={alignment}
+            {...styles('root', { mobile, alignment }, rest)}
+            {...getDataAttributes(mobile)}
+            role="menu"
+            tabIndex={-1}
+          >
+            {React.Children.map(children, (child: React.ReactElement) =>
+              child.type === ActionsMenuItem
+                ? React.cloneElement(child, { className: styles.item })
+                : React.cloneElement(child, { className: styles.divider }),
```

Comment:
I actually think that passing the classNames aren't really necessary here.
I think the static Divider can be a component that wraps the `wix-ui-tpa/Divider`, and adds the necessary padding.

Reply:
What about this?

Reply:
resolved